praw/reddit.py

```diff
@@ -579,10 +607,41 @@ def post(
             None).
 
         """
-        data = self.request(
-            "POST", path, data=data or {}, files=files, params=params
-        )
-        return self._objector.objectify(data)
+        retries = 0
+        while True:
+            _raise = True
+            try:
+                return self._objectify_request(
+                    "POST", path, data=data or {}, files=files, params=params
+                )
+            except RedditAPIException as exception:
+                print(exception)
+                for item in exception.items:
+                    if "ratelimit" in item.error_type.lower():
```

Comment:
Let's drop the lower and match Reddit's casing unless we have an example where Reddit is inconsistent.